history/subscriber/persister.py

```diff
@@ -289,8 +288,10 @@ def start_dojot_messenger(config, persister):
                  "message", persister.handle_new_tenant)
     messenger.on("dojot.notifications", "message",
                  persister.handle_notification)
+    LOGGER.info('Listen to notification events')
```

Comment:
Note that tenant events are being listened too. So, add `LOGGER.info("Listen to tenancy events")`